plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -330,7 +313,7 @@ def test_get_ion_density_OML_without_return_fit():
 
     char = characteristic()
     density = langmuir.get_ion_density_OML(char, 5000000*u.m**2,
-                                           1*u.u, return_fit=False)
+                                           'p+', return_fit=False)
     assert np.isclose(density.value, 383949768.0764418)
```

Comment:
I did also have to update this test value to `385344135.12064785`, which is a bit strange, but given that this was a regression test... no, actually, I'm not quite sure why this would change.

Reply:
The mass of a proton is about 1.007 u, so we're using a different mass.  The definition of an atomic mass unit is based on 1/12 of the mass of a neutral atom of carbon-12.

Reply:
Okay, I guess I wasn't thinking clearly then. Too much multitasking is not good for your code reviews!

Reply:
Ah makes sense, going to do that as soon as I get home! I'm still new to tests so thanks for all the guidance so far!